demo_umami.install

```diff
@@ -72,4 +95,7 @@ function demo_umami_install() {
   // in the demo_umami.info.yml file, as it requires configuration provided by
   // the profile (fields etc.).
   \Drupal::service('module_installer')->install(['demo_umami_content'], TRUE);
+
+  // Storing Drupal version on demo_umami install.
```

Comment:
I'd possibly change this comment to:

// Store the version of Drupal installed.